M2/Macaulay2/d/binding.d

```diff
@@ -311,6 +311,7 @@ bumpPrecedence();
      export elapsedTimeS := special("elapsedTime",unaryop,precSpace,wide);
      export elapsedTimingS := special("elapsedTiming",unaryop,precSpace,wide);
      export shieldS := special("shield",unaryop,precSpace,wide);
+     export testS := special("TEST",unaryop,precSpace,wide);
```

Comment:
I'm a little unsure if `testS` is the right term here, especially if we wanted an actual `test` keyword in the future. `TESTS` also seems wrong, but maybe `TestS`?

Reply:
Sounds good -- I'll switch it to `TestS`